contracts/FlowIDTableStaking.cdc

```diff
@@ -921,23 +913,31 @@ pub contract FlowIDTableStaking {
             FlowIDTableStaking.account.save(slotLimits, to: /storage/flowStakingSlotLimits)
         }
 
+        /// Sets the number of open node slots to allow per epoch
+        /// Only access nodes are used for this currently,
+        /// but other node types will be added in the future
+        pub fun setOpenNodeSlots(openSlots: {UInt8: UInt16}) {
+            pre {
+                openSlots[UInt8(5)] != nil: "Need to have a value set for access nodes"
```

Comment:
What is the magic number 5 here?

Reply:
It is the access node role number. We can't add fields in the contract, so we just use the magic number